uint/src/uint.rs

```diff
@@ -38,6 +38,21 @@ pub enum FromDecStrErr {
 	InvalidLength,
 }
 
+impl std::fmt::Display for FromDecStrErr {
```

Comment:
the CI failure is legit, we should hide this (and the one below) impl under `cfg(feature = std)`
https://github.com/paritytech/parity-common/blob/46d8f6b5419edea4c7b2d46a09ecf3bee3facc7b/uint/src/uint.rs#L1546